### DISCUSSION TITLE
Inductive probes

### DISCUSSION THREAD

**Discussion by supersonical** [Jyers/Marlin#297] [Ideas]
Personally i feel that inductive probes is superior to blt touch probs but it seems no one is compiling firmware for it which is a shame. This makes it much steper learning curve for beginners to have inductive probes as a option to the blt touch. 
And to be honest, i´m a noob aswell and have managed to hack it together whith jyers source code to get it working but its far from what it could be if somebody with knowledge could compile jyers code correctly for inductive probes. 
What do you guys think?

**Comment by MarcWinNJ** [comment]
I was just looking through the repository for code for an ABL probe.I run an Autolev sensor.  Also need one for a direct drive thermistor setup, V6 hot end
